### PR TITLE
Add Calculator Components

### PR DESCRIPTION
**Pull request by Ibnballo1**
I added the following components to calculator app:

- The result component: This is the component where the result from the calculation will be shown
- The Digit component: This is where the digits are shown
- The operator component: This is where the operators are displayed

### CODE REVIEW

**Comment by Ibnballo1** [comment]
Thanks @SajjadAhmad14 , please how can I solve this issue... ?
I don't know that having a development branch is crucial for this project
.

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,13 @@
-import logo from './logo.svg';
+/* eslint-disable linebreak-style */
+/* eslint-disable react/self-closing-comp */
+/* eslint-disable linebreak-style */
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (ShadyShawkat) [inline src/App.js:7]: - [ ] Kindly make sure that all of your components are class components, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=(All%20components%20should%20be%20class%2Dbased)) requirement.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        <div className="eachSigns">
+          <p className="signs">+</p>
+        </div>
+        <div className="eachSigns">
+          <p className="signs">=</p>
+        </div>
+      </div>
+    );
+  }
+}
```
Comment (ShadyShawkat) [inline src/components/Calculator.js:105]: - [ ] [OPTIONAL] You can consider moving these components out of this file into their own UI components, and importing them here, as a best practice.

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Ibnballo1 👋, 

Great work on making the changes requested by a previous reviewer 👏 

 ## Highlights
- The use of proper Gitflow ✔️ 
- Styling of the calculator matches that of the template given perfectly ✔️ 
- Use of `class-based` React components ✔️ 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
#### I am making this comment in the review section because I can not highlight the code in the README.md file which is not in the files change section.

- [ ] Kindly make changes to your `README.md` file. You can use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) as a guide. Please include instructions for:
- Cloning the repository `git clone ...`
- Installing all dependencies `npm install`
- Creating a build directory with a production build of your app `npm run build`
- Running the dev server `npm start`

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ibnballo1** [comment]
Thanks @Gambit142 , but I don't think this should make you not approved the project

**Review by Wes-Isaac — APPROVED** [review]
Hi @Ibnballo1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- README looks professional👀.
- Project works as expected 🚀.
- Linters are all green ✔️.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**